modules/nw-operator-cr.adoc

```diff
@@ -419,24 +399,25 @@ kind: Network
 metadata:
   name: cluster
 spec:
-  clusterNetwork: <1>
-  - cidr: 10.128.0.0/14
+  clusterNetwork: 
+  - cidr: 172.24.0.0/14
```

Comment:
why this changes?

Reply:
yeah, you should leave this as it was; `10.128.0.0/14` is the default clusterNetwork value for OCP

Reply:
I reverted these changes. Thanks!